services/UserService.py

```diff
@@ -1,12 +1,55 @@
+import string
+from flask import current_app as app
+
+from dtos import UserDto
+from dtos.errors import BadRequestError
 from repositories.UserRepository import UserRepository
+from dtos.responses import DataResponse, DeleteResponse, CreateResponse
+
+
+def validateNewUser(user: UserDto):
+    badVal = None
+    if user["username"] is None or len(user["username"]) == 0:
+        badVal = "username"
+
+    if user["password"] is None:
+        badVal = "password"
+
+    if user["email"] is None:
+        badVal = "email"
+
+    if user["name"] is None:
```

Comment:
if `__getattr__()` is committed:
```suggestion
    if user.username is None or len(user.username) == 0:
        badVal = "username"

    if user.password is None:
        badVal = "password"

    if user.email is None:
        badVal = "email"

    if user.name is None:
```